tests/integration/dlc/dlc.test.ts

```diff
@@ -359,6 +370,26 @@ describe('dlc provider', () => {
         ).to.be.eventually.rejectedWith(Error);
       });
 
+      it('should fail verify batch close with fixedInputs', async () => {
```

Comment:
Do we have a test for invalid signatures?

What if we modify `offerPayoutSatoshis` / `acceptPayoutSatoshis` to something invalid?

Reply:
Great point, added a new test for this: https://github.com/AtomicFinance/chainabstractionlayer-finance/pull/92/files#diff-1da501fc80c3eea9a0e1e9c1e9198f77993e80cb9650dfc96da767919ca18abaR407